.github/SECURITY.md

```diff
@@ -0,0 +1,3 @@
+# Django Security Policies
+
+Please see https://www.djangoproject.com/security/ .
```

Comment:
👀 
```suggestion
Please see https://www.djangoproject.com/security/.
```

Reply:
I left a space because some naive link detectors grab trailing punctuation like dots, but I guess this is an old habit. It seems Chrome and Firefox at least strip trailing dots on links.

Reply:
Yeah, most parsers are ignoring punctuation out of experience at least it's been years now since I noticed one including punctuation in the URLs. But GitHub's never has issues with parsing URLs properly.